store/storetest/post_store.go

```diff
@@ -1391,7 +1391,7 @@ func testPostStoreGetFlaggedPostsForTeam(t *testing.T, ss store.Store, s SqlSupp
 
 	store.Must(ss.Preference().Save(&preferences))
 
-	r2 := (<-ss.Post().GetFlaggedPostsForTeam(o1.UserId, c1.TeamId, 0, 2)).Data.(*model.PostList)
+	r2, _ := ss.Post().GetFlaggedPostsForTeam(o1.UserId, c1.TeamId, 0, 2)
```

Comment:
Would you please add two checks:
- `require.NotNil(t, r2)`
- `require.NotError(t,err)` for the currently ignore return value

Reply:
The pattern that we are using for this kind of things is use `require.Nil(t, err)`. I think "NotError" given use certain problems in the past. And normally we don't check the `require.NotNil` for the result because if the error is nil, is because the result is not nil.